README.md

```diff
@@ -1,207 +1,232 @@
 # SignalFx Java Agent
 
-This is the SignalFx Java Agent, a [JVM
-agent](https://docs.oracle.com/javase/7/docs/api/java/lang/instrument/package-summary.html)
-to automatically instrument your Java application to capture and report
-distributed traces to SignalFx. Enabling the agent is done by downloading the
-[latest JAR](https://github.com/signalfx/signalfx-java-tracing/releases/latest)
-and adding its path to your JVM startup options:
-
-```bash
-$ # Specify the Java Agent with the appropriate option and launch your application as usual
-$ java -javaagent:signalfx-tracing.jar -jar app.jar
-```
+The SignalFx Java Agent is a
+[Java Virtual Machine](https://docs.oracle.com/javase/7/docs/api/java/lang/instrument/package-summary.html)
+(JVM) agent that automatically instruments your Java application to capture and
+report distributed traces to SignalFx. Download the JAR for the agent's 
+[latest release](https://github.com/signalfx/signalfx-java-tracing/releases/latest)
+and add it to your JVM path.
```

Comment:
I wouldn't change the existing startup options language or preferably remove the `-javaagent` example at the beginning since it clearly demonstrates the simplicity of use.

Reply:
We can include it at the top and then link to the section for more information.